src/ai/backend/manager/services/group/types.py

```diff
@@ -107,8 +107,6 @@ def fields_to_update(self) -> dict[str, Any]:
         self.is_active.update_dict(to_update, "is_active")
         self.domain_name.update_dict(to_update, "domain_name")
         self.total_resource_slots.update_dict(to_update, "total_resource_slots")
-        self.user_update_mode.update_dict(to_update, "user_update_mode")
-        self.user_uuids.update_dict(to_update, "user_uuids")
```

Comment:
If it's not a modifiable value, could you take it out of the modifier and put it in the action?

Reply:
It is not modifiable in groups table but it can update the association table `association_groups_users` which associates groups and users table. Should I consider the `...Modifier` types as DB table updaters?

Reply:
We agreed to set `...Modifier` types as DB table updaters.